packages/augur-ui/src/modules/portfolio/components/markets/markets.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Link } from 'react-router-dom';
+import { Link, Redirect } from 'react-router-dom';
```

Comment:
get rid oft his import since I don't think it's used.

Reply:
Done